src/injectedScript.ts

```diff
@@ -185,7 +184,7 @@ async function signExtrinsicWithDid(
 
 async function getSignedDidCreationExtrinsic(
   submitter: KiltAddress,
-  pendingDidUri?: DidUri,
+  pendingDidUri?: Did,
```

Comment:
Also here.

Reply:
I'm not concerned with changing any of the naming in this PR. It's just to test how to alpha works with Sporran and will not be merged into the main branch